packages/block-editor/src/components/colors-gradients/panel-color-gradient-settings.js

```diff
@@ -99,8 +42,9 @@ export const PanelColorGradientSettingsInner = ( {
 	__experimentalHasMultipleOrigins,
 	__experimentalIsRenderedInSidebar,
 	enableAlpha,
-	...props
 } ) => {
+	const panelId = useInstanceId( PanelColorGradientSettingsInner )
```

Comment:
```suggestion
	const panelId = useInstanceId( PanelColorGradientSettingsInner );
```

My understanding is that the `panelId` is only needed for rendering via slot fills into the `ToolsPanel`. We could probably leave it out here given the `PanelColorGradientSettings` panels aren't leveraging slots.